packages/bot-web-ui/src/components/dashboard/dashboard-components/index.scss

```diff
@@ -0,0 +1,166 @@
+.bot-dashboard {
+    .toolbar__section {
+        justify-content: end;
+    }
+    .run-panel__container {
+        height: calc(100vh - 15rem) !important;
+        position: unset !important;
+        transform: unset !important;
+        margin-right: -350px;
+        transition: all 0.4s;
+        margin-top: 48px;
+    }
+    .dc-drawer--open {
+        margin-right: 0 !important;
+    }
+}
+.dashboard {
+    &__container {
+        border: none;
+        max-width: 100%;
+        // border: solid 1px var(--general-section-1);
+        background: #fff;
```

Comment:
Check all the colors in your PR and use the variables that we already have

Reply:
done